### PR TITLE
 LRU_Cache.java

### PR DESCRIPTION
**Pull request by Sakshamgupta90**
Please review this PR

### CODE REVIEW

**Comment by Pradyuman7** [comment]
@Sakshamgupta90 please make sure the build passes for your PR, you can check the log to get more details regarding the failure, but here's a small part from it:
```
LeetCodeSolutions: Compilation failure
Error:  /home/runner/work/Let-Us-Java/Let-Us-Java/LeetCodeSolutions/src/main/java/LRU_Cache.java:[5,1] class, interface, or enum expected
```

**Comment by Sakshamgupta90** [comment]
@Pradyuman7 Sorry I didnt understand what's the problem can you please clarify!

File: LeetCodeSolutions/src/main/java/LRU_Cache.java [hunk 1]
```diff
@@ -0,0 +1,153 @@
+// Leetcode Ques No. : 146
+// Problem Link: https://leetcode.com/problems/lru-cache/
+
+
+***** problem statement *****
```
Comment (Pradyuman7) [inline LeetCodeSolutions/src/main/java/LRU_Cache.java:5]: This should not be here like this (add it as a comment or remove this)

File: LeetCodeSolutions/src/main/java/LRU_Cache.java [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+Design a data structure that follows the constraints of a Least Recently Used (LRU) cache.
+Implement the LRUCache class:
+LRUCache(int capacity) Initialize the LRU cache with positive size capacity.
+int get(int key) Return the value of the key if the key exists, otherwise return -1.
+void put(int key, int value) Update the value of the key if the key exists. Otherwise, add the key-value pair to the cache. 
+If the number of keys exceeds the capacity from this operation, evict the least recently used key.
+*/
+
+
+***** Solution *****
```
Comment (Pradyuman7) [inline LeetCodeSolutions/src/main/java/LRU_Cache.java:17]: Same for this

File: LeetCodeSolutions/src/main/java/LRU_Cache.java [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+
+
+ import java.util.HashMap;
+
+ public class LRUCache {
```
Comment (Pradyuman7) [inline LeetCodeSolutions/src/main/java/LRU_Cache.java:23]: File name and public class name should be the same

**Review by Pradyuman7 — CHANGES_REQUESTED** [review]

**Comment by Sakshamgupta90** [comment]
@Pradyuman7 now is this fine?

**Review by Pradyuman7 — APPROVED** [review]

**Comment by Sakshamgupta90** [comment]
@Pradyuman7 can you please add Hacktoberfest accepted tag in this.

**Pull request merged**